pyipn/geometry.py

```diff
@@ -51,6 +53,11 @@ def coord(self):
     def get_cartesian_coord(self):
         return self._skycoord.represent_as(CartesianRepresentation)
 
+    def get_norm_vec(self, unit):
```

Comment:
we should add assertions here to make sure a 'unit' is passed. 

```python
assert isinstance(unit, satrapy.units.Unit)

```

or something like this